redux-demo/src/CommentForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './CommentForm.css';
+
+const CommentForm = add =>
+    <form>
+        <div>
+            <label htmlFor="new_comment">Add a comment:</label>
+            <textarea id="new_comment" name="new_comment" rows="6"></textarea>
+        </div>
+        <div>
+            <button className="btn-add" onClick={(e) => {e.preventDefault();
+                const form = document.getElementById("new_comment");
```

Comment:
możesz skorzystać z `ref` zamiast wyszukiwać element - https://reactjs.org/docs/refs-and-the-dom.html

Reply:
a jeszcze lepiej będzie modyfikować state komponentu - https://reactjs.org/docs/forms.html#the-textarea-tag i tam przechowywać value